app/views/clients/_tab_navigation.haml

```diff
@@ -22,14 +26,15 @@
       permission: show_health
     }
   }
-  
-%ul.nav.nav-tabs.margin-bottom-01
-  - tabs.each do |action, details|
-    - if action == current
-      - if details[:permission]
-        %li.active.active-tab{role: 'presentation'}
-          %a{href: '#'} 
-            = details[:title]
-    - else
-      - if details[:permission]
-        %li= link_to details[:title], action
+
+- if can_edit_clients? || can_manage_client_files? || show_health
```

Comment:
@everestwitman can you move this set of checks into a method on GrdaWarehouse::Hud::Client that is called `def self.should_see_client_dashboard_navigation`?